content/control-api/testing-with-postman.textile

```diff
@@ -12,73 +12,83 @@ jump_to:
+  <img src="/images/screenshots/control-api/postman-import-dialog.png" style="width: 100%" alt="Postman import dialog">
 </a>
 
-3. Drag and drop the OAS3 document you want to test, or use the "Upload file" button to select and import an OAS3 document. Alternatively, if you haven't already downloaded the document, you can provide a link to it in the Ably OpenAPI documents "GitHub repository":https://github.com/ably/open-specs/definitions/control-v1.yaml.
+3. Click the "Link" tab, and paste in the following URL: @https://raw.githubusercontent.com/ably/open-specs/main/definitions/control-v1.yaml@, then click "Continue":
 
-4. In the Import dialog use the default settings and click "Import".
+<a href="/images/screenshots/control-api/postman-link.png" target="_blank">
+  <img src="/images/screenshots/control-api/postman-link.png" style="width: 100%" alt="Link to OpenAPI document">
+</a>
+
+4. In the Import dialog, use the default settings and click "Import":
 
-5. The imported document will now appear in your Postman collections:
+<a href="/images/screenshots/control-api/postman-import-default.png" target="_blank">
+  <img src="/images/screenshots/control-api/postman-import-default.png" style="width: 100%" alt="Postman import default">
+</a>
+
+5. The imported document now appears in your Postman collections:
 
 <a href="/images/screenshots/control-api/imported-collection.png" target="_blank">
-  <img src="/images/screenshots/control-api/imported-collection.png" style="width: 100%" alt="Postman imported collection">
+  <img src="/images/screenshots/control-api/imported-collection.png" style="width: 80%" alt="Postman imported collection">
 </a>
 
-You are now ready to test out the API.
+You have now imported the OpenAPI document into Postman. 
 
 h2(#send-request). Sending a request
 
-You have now imported the OAS3 document into Postman. You now need to "configure a token":#authentication to authenticate any requests you make to the Control API.
+First, you need to "obtain a Control API token":/control-api#creating-access-token to authenticate any requests you send to the Control API.
 
-For many Control API requests you will also need to specify your Account ID or Application ID. You can see the "documentation":/control-api#ids for details on how to obtain these.
+For many Control API requests you also need to specify your account ID or app ID. You can see the "account ID and app ID":/control-api#ids section for details on how to obtain these.
 
-Now that you understand where to obtain these items, you will learn how to create and execute a request in Postman. 
+Now that you have obtained the token and IDs, you can learn how to send a request in Postman. 
 
-1. The following screenshot shows the required parameters for the "List account apps" request:
+1. Expand the collection to the @List account apps@ request. The following screenshot shows the parameters for the request:
```

Comment:
FYI: This is actually '_Lists_ account apps' in the Postman collection.

Reply:
Well spotted! Thanks for the quick review.